src/models/package.js

```diff
@@ -4,8 +4,12 @@ const { Schema } = mongoose;
 const schema = new Schema({
     name: { type: String, required: true, trim: true},
     volume: Number, 
-    recolectionPeriod: {type: Number, min: 1, max: 2},
-    price: Number,
+    pickupPeriod: {type: Number, min: 1, max: 2},
+    fullPrice: Number,
```

Comment:
is this working like this? i would love to see a demo.

Reply:
sure